static/src/javascripts-legacy/projects/commercial/modules/survey/survey-simple.js

```diff
@@ -1,6 +1,6 @@
 define([
-    'bean',
-    'fastdom',
+    'Promise',
```

Comment:
Ok, maybe that's because it's still in `javascripts/legacy`. Never mind - soon we'll be able to remove it!